narayana-integration-bom/pom.xml

```diff
@@ -40,18 +40,18 @@
         <groupId>org.jboss.jbossts.jta</groupId>
         <artifactId>narayana-jta</artifactId>
         <version>${version.org.jboss.jbossts.jta}</version>
-        <exclusions>
```

Comment:
This was here by mistake I think. As there's no dependency on this on this `narayana-jta`.

Reply:
maybe it was there as transitive dependency in past